packages/Utils/hex-to-rgba.ts

```diff
@@ -1,8 +1,14 @@
 import { hexToRGB } from './hex-to-rgb'
 
+const colorFormatRegex =
+  /^(rgb|hsl)(a?)[(]\s*([\d.]+\s*%?)\s*,\s*([\d.]+\s*%?)\s*,\s*([\d.]+\s*%?)\s*(?:,\s*([\d.]+)\s*)?[)]$/
```

Comment:
This regex doesn't match most of the [newish colors notations](https://caniuse.com/mdn-css_types_color_rgba_space_separated_parameters)